src/plugins/intel_cpu/src/nodes/executors/jit/eltwise.cpp

```diff
@@ -0,0 +1,608 @@
+                                                                       ov::element::u8};
+    const auto& supported_output_precisions = supported_input_precisions;
+#endif
+
+#if defined(OPENVINO_ARCH_ARM64) || defined(OPENVINO_ARCH_RISCV64)
+    const auto check_precisions = [&](const std::vector<ov::element::Type>& input_precisions,
+                                      const std::vector<ov::element::Type>& output_precisions) {
+        if (std::any_of(input_precisions.begin(),
+                        input_precisions.end(),
+                        [&supported_input_precisions](const ov::element::Type& precision) {
+                            return !contains(supported_input_precisions, precision);
+                        })) {
+            return false;
+        }
+
+        if (std::any_of(output_precisions.begin(),
+                        output_precisions.end(),
+                        [&supported_output_precisions](const ov::element::Type& precision) {
+                            return !contains(supported_output_precisions, precision);
+                        })) {
+            return false;
+        }
+
+        return true;
+    };
+
+    return check_precisions(input_precisions, output_precisions);
+#endif
+
+    // Unsupported architectures should return false:
+    return false;
+}
+
+bool EltwiseJitExecutor::supports(const EltwiseConfig& config) {
+    std::vector<ov::element::Type> input_precisions(config.descs.size() - 1);  // -1 for output precision
+    std::vector<ov::element::Type> output_precisions{
+        config.descs.at(ARG_DST)->getPrecision()};  // -1 for output precision
+
+    for (const auto& [argId, desc] : config.descs) {
+        if (argId == ARG_DST) {
+            continue;  // Skip output precision
+        }
+
+        input_precisions[argId - ARG_SRC] = desc->getPrecision();
+    }
+
+    return supports(config.attrs, srcRank(config), input_precisions, output_precisions);
+}
+
+const VectorDims& EltwiseJitExecutor::getOutDims() const {
+    if (!m_kernel) {
+        OPENVINO_THROW("Can't get jit eltwise params, kernel for Eltwise executor is not compiled");
+    }
+    return m_kernel->jep_.dims;
+}
+
+size_t EltwiseJitExecutor::getBatchDimIdx() const {
+    return m_batchDimIdx;
+}
+
+impl_desc_type EltwiseJitExecutor::implType() const {
+#if defined(OPENVINO_ARCH_ARM64)
+    if (mayiuse(dnnl::impl::cpu::aarch64::asimd)) {
+        return impl_desc_type::jit_asimd;
+    }
+#elif defined(OPENVINO_ARCH_RISCV64)
+    if (mayiuse(ov::intel_cpu::riscv64::gv)) {
+        return impl_desc_type::jit_gv;
+    }
+#elif defined(OPENVINO_ARCH_X86_64)
+    if (mayiuse(dnnl::impl::cpu::x64::avx512_core)) {
+        return impl_desc_type::jit_avx512;
+    }
+
+    if (mayiuse(dnnl::impl::cpu::x64::avx2)) {
+        return impl_desc_type::jit_avx2;
+    }
+
+    if (mayiuse(dnnl::impl::cpu::x64::sse41)) {
+        return impl_desc_type::jit_sse42;
+    }
+#endif
+    return impl_desc_type::ref;
```

Comment:
Just to double check:
Is it okey that `EltwiseJitExecutor::implType` might return `ref`? Or should be there exception if we achieve this line?